src/components/Home.tsx

```diff
@@ -95,7 +104,7 @@ const Home: React.FC = () => {
                 amount: numberedAmount,
                 categoryId: categories[categoryIdx]?.id ?? 0,
                 fromAccountId: accounts[accountIdx]?.id ?? 0,
-                description: "-",
+                title,
```

Comment:
Why do you change from description to title? I think description is better

Reply:
Let's discuss the title field in Home. But in More, I rename description to title because, although a user is filling in the description, I want it to be a short one (TextBox instead of TextArea). So I change the label from description to title to make it feel like a short title which show on each transaction.